docs/modules/ROOT/pages/hazelcast-platform-operator-wan-replication.adoc

```diff
@@ -416,7 +442,7 @@ You should see the following output:
 [source, shell]
 ----
 Successful connection!
-Current map size: 12
+Current map (<MAP-NAME>) size: 12
 ----
 
 == Clean Up
```

Comment:
Suggested updates to line 424:

To remove all custom resources and PVCs, run the following commands: